sdks/python/apache_beam/coders/coder_impl.py

```diff
@@ -97,7 +98,9 @@ def get_estimated_size_and_observables(self, value, nested=False):
 
 
 class SimpleCoderImpl(CoderImpl):
-  """Subclass of CoderImpl implementing stream methods using encode/decode."""
+  """For internal use only; no backwards-compatibility guarantees.
```

Comment:
So the thinking was that if the class should not be used by users, it might be good to have this as a summary. Also, I generated a doc using pydoc and this doesn't seem to be being rendered in a special way at least for that tool.